examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java

```diff
@@ -13,9 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.jagrosh.jdautilities.command.examples;
+package com.jagrosh.jdautilities.examples.command;
```

Comment:
this package renaming is redundant?
If it is, we should promptly change it back

Reply:
I've done this because I moved it to it's own `examples` module. The idea was to keep all examples in one place, including future examples for e.g. the module system. If you think this isn't a good idea or there probably won't be enough examples to justify this I can just put them back into their old package and into the `command` module.